src/create_torrent.cpp

```diff
@@ -75,7 +76,7 @@ namespace libtorrent
 			return 0;
 #else
 			struct stat s;
-			if (lstat(convert_to_native(p).c_str(), &s) < 0) return 0;
+			if (lstat(convert_to_native(std::string(p)).c_str(), &s) < 0) return 0;
```

Comment:
yes :) and you already use it https://github.com/arvidn/libtorrent/blame/master/src/kademlia/node.cpp#L779